app/(app)/feature-flag-example/_client.tsx

```diff
@@ -1,24 +1,20 @@
 "use client";
+
```

Comment:
This file was meant to be kept as an example. I don't think I was clear enough in the issue so that's my fault. But it is meant to be in `app/(app)/courses` so when a user visits `/courses` they can find this page.